swap/swap_test.go

```diff
@@ -220,26 +219,22 @@ func TestStoreBalances(t *testing.T) {
 	s, clean := newTestSwap(t, ownerKey)
 	defer clean()
 
-	var err error
-
-	// modify balances in memory but not in store
-	testPeerID := enode.HexID("8418f4eeb20735630cfff00459b7fc7ec1674c9f77f19bab23e895706bfd1032")
-	s.balances[testPeerID] = 144
-	comparePeerBalance(t, s, testPeerID, 0)
-
 	// modify balances both in memory and in store
-	peerBalance, err := s.updateBalance(testPeerID, 29)
-	if err != nil {
+	testPeer := s.newPeer(newDummyPeer().Peer, common.Address{}, common.Address{})
+	testPeerID := testPeer.ID()
+	var peerBalance int64 = 29
```

Comment:
In this case use this notation: `peerBalance := int64(29)`

Reply:
done